dev/integration_tests/ui/pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: integration_ui
+description: Flutter non-plugin UI integration tests.
+
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_driver:
+    sdk: flutter
+dev_dependencies:
+  test: 0.12.20
```

Comment:
you can make this "any" if you add a dependency to flutter_test, that way it's easier for us to keep them in sync.